app/recordtransfer/templates/recordtransfer/email/submission_success.html

```diff
@@ -14,17 +14,19 @@
     </p>
     <p class="main-content">
         {% blocktrans %}
-        If you have any questions about your submission, please contact an NCTR archivist
-        at this email address:
+            You can view your past submissions by clicking on the button below:
         {% endblocktrans %}
     </p>
     <div class="main-content">
-        <a href="mailto:{{ archivist_email }}" class="email-btn">{% blocktrans %}Contact NCTR Archivist{% endblocktrans %}</a>
+        {% url 'recordtransfer:user_profile' as user_profile_url %}
+        <a href="https://{{ site_domain }}{{ user_profile_url }}#id_past_submissions_tab"
```

Comment:
Going to this link doesn't select this tab in the table when going to this link, is there an easy way to make that happen?